tests/ttnn/integration_tests/resnet/test_ttnn_functional_resnet50_new.py

```diff
@@ -279,7 +279,7 @@ def create_test_infra(device, batch_size, act_dtype, weight_dtype, math_fidelity
         # (20, ttnn.bfloat8_b, ttnn.bfloat8_b, ttnn.MathFidelity.LoFi),
     ),
 )
-def test_resnet_50(device, batch_size, act_dtype, weight_dtype, math_fidelity):
+def test_resnet_50(device, use_program_cache, batch_size, act_dtype, weight_dtype, math_fidelity):
```

Comment:
`use_program_cache` is not used in the function?

Reply:
It's a fixture. It sets up program caching behind the scenes